pkgs/applications/audio/lv2lint/default.nix

```diff
@@ -1,23 +1,31 @@
-{ stdenv, lib, fetchurl, pkg-config, meson, ninja, lv2, lilv, curl, libelf }:
+{ stdenv, lib, fetchgit, pkg-config, meson, ninja, lv2, lilv, curl, elfutils, xorg }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "lv2lint";
   version = "0.16.2";
 
-  src = fetchurl {
-    url = "https://git.open-music-kontrollers.ch/lv2/${pname}/snapshot/${pname}-${version}.tar.xz";
-    sha256 = "sha256-sjgQVx8uGNPWcUwKzGUhChpfzXj/8D8cggVTpcHEXPQ=";
+  src = fetchgit {
+    url = "https://git.open-music-kontrollers.ch/~hp/lv2lint";
+    rev = finalAttrs.version;
+    hash = "sha256-NkzbKteLZ+P+Py+CMOYYipvu6psDslWnM1MAV1XB0TM=";
   };
```

Comment:
please use `fetchFromSourcehut`
```nix
  src = fetchFromSourcehut {
    domain = "open-music-kontrollers.ch";
    owner = "~hp";
    repo = "lv2lint";
    rev = finalAttrs.version;
    hash = "sha256-NkzbKteLZ+P+Py+CMOYYipvu6psDslWnM1MAV1XB0TM=";
  };
```

Reply:
Can do; curious as to why a more specific (and therefore likely to break) source fetcher is preferred to a more general one like fetchgit.

Reply:
I've switched to `fetchFromSourcehut`. Still sort of skeptical about whether using a more specific fetcher has any real benefits.